public/script/poll.js

```diff
@@ -1,6 +1,6 @@
 (function() {
 
-    var pageId = window.location.pathname.split('/')[3];
+    var pageId = document.getElementById('pollId').textContent;
```

Comment:
wont this break the poll page since the route expects id through url?

Reply:
tested this, i had to pass a pollid to poll.mustache. It should work now